pyface/ui/qt4/timer/do_later.py

```diff
@@ -13,4 +13,4 @@
 Provided for backward compatibility.
 """
 
-from pyface.timer.do_later import DoLaterTimer
+from pyface.timer.do_later import DoLaterTimer  # noqa: F401
```

Comment:
Should this module raise a `DeprecationWarning`?

Reply:
_and another issue needed regarding deprecations_

Reply:
i opened - https://github.com/enthought/pyface/issues/916